common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -795,6 +797,61 @@ def test_courseware_nav(self):
         self.course_nav.go_to_section('Test Section 2', 'Test Subsection 3')
         self.assertTrue(self.course_nav.is_on_section('Test Section 2', 'Test Subsection 3'))
 
+    # def test_course_home(self):
+    #     """
+    #     Navigate to the course home page using the tab.
+    #     """
+    #     # TODO: TNL-6546: Remove override_flag.
+    #     from waffle.testutils import override_flag
+    #     # ERROR: Import results in the following error:
+    #     # - ImproperlyConfigured: Requested setting CACHES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
+    #     with override_flag('unified_course_view', active=True):
+    #         self.course_info_page.visit()
+    #         self.tab_nav.go_to_tab('Course')
+    #         course_home = CourseHomePage(self.browser, self.course_id)
+    #
+    #         course_home.is_browser_on_page()
+
+    def test_course_home(self):
+        """
+        Navigate to the course home page using the tab.
+        """
+        # TODO: TNL-6546: Use tab navigation and remove course_home.visit().
+        #self.course_info_page.visit()
+        #self.tab_nav.go_to_tab('Course')
+        course_home = CourseHomePage(self.browser, self.course_id)
+        course_home.visit()
+
+# TODO: Move this to a separate CourseHomeTests
```

Comment:
Because this was WIP, this TODO was a note to self.  I wasn't expecting to merge.  I want to find a nice home for this CourseHome smoke test, and the a11y test, etc.  I'll provide an update soon I hope.